cmd/rekor_consistency/config.go

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright 2024 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"time"
+
+	"github.com/sigstore/rekor-monitor/pkg/identity"
+)
+
+type ConsistencyCheckConfiguration struct {
```

Comment:
What do you think about omitting this and sticking to CLI flags? I think that yaml is great for complex input or for server configuration, but since we expect this to be run as a tool, these can be flags.

Given we also have other flags already, I don't think we need a configuration too.

If we remove this, then we can have a `monitored-values` flag as a string and a `monitored-value-file` flag as a path to a file.

Reply:
Do we want to address the last part of this:

> If we remove this, then we can have a monitored-values flag as a string and a monitored-value-file flag as a path to a file.

Adding a flag for passing by file as well?